SR2023/2023-01-19-competition-venue-announcement.md

```diff
@@ -0,0 +1,35 @@
+---
+to: Student Robotics 2023 teams
+subject: SR2023 Competition Venue
+---
+
+Hello!
+
+We're thrilled to announce the venue for Student Robotics 2023!
```

Comment:
Maybe add a heading here so it's balanced with regards to the other sections?

Reply:
https://github.com/srobo/team-emails/pull/136/commits/ac993b1c30c3f4bca44dd1cc8190ab752946ca4e